doc/classes/DisplayServer.xml

```diff
@@ -984,6 +984,8 @@
 				- [code]name[/code] is voice name.
 				- [code]id[/code] is voice identifier.
 				- [code]language[/code] is language code in [code]lang_Variant[/code] format. [code]lang[/code] part is a 2 or 3-letter code based on the ISO-639 standard, in lowercase. And [code]Variant[/code] part is an engine dependent string describing country, region or/and dialect.
+				Note that, on Linux, Godot depends on system-dependent libraries for text-to-speech functionality. If they are not present, this method will return an empty list. This applies to both Godot users on Linux, as well as end-users on Linux running Godot games that use text-to-speech.
```

Comment:
I would suggest changing "system-dependent libraries" to "system libraries" for clarity.